README.md

```diff
@@ -39,44 +42,59 @@ curl -fsSL https://pkgs.zabbly.com/key.asc | sudo gpg --dearmor -o /etc/apt/keyr
 
 ### Stable repository
 
-On any of the distributions above, you can add the package repository at `/etc/apt/sources.list.d/zabbly-incus-stable.sources`:
+On any of the distributions above, you can add the package repository at
+`/etc/apt/sources.list.d/zabbly-incus-stable.sources`.
 
-```
+Run the following command to add the stable repository:
+
+```sh
+sudo bash -c 'cat <<EOF > /etc/apt/sources.list.d/zabbly-incus-stable.sources
 Enabled: yes
 Types: deb
 URIs: https://pkgs.zabbly.com/incus/stable
-Suites: DISTRO
+Suites: $(lsb_release -sc)
 Components: main
-Architectures: ARCH
-Signed-By: /etc/apt/keyrings/zabbly.asc
-```
+Architectures: $(dpkg --print-architecture)
+Signed-By: /etc/apt/keyrings/zabbly.gpg
 
-Make sure to replace `DISTRO` with one of `focal`, `jammy`, `bullseye` or `bookworm`
-and then replace `ARCH` with one of `amd64` or `arm64`.
+EOF'
+```
 
 ### Daily repository
 
-On any of the distributions above, you can add the package repository at `/etc/apt/sources.list.d/zabbly-incus-daily.sources`:
+On any of the distributions above, you can add the package repository at
+`/etc/apt/sources.list.d/zabbly-incus-daily.sources`.
 
-```
+Run the following command to add the daily repository:
+
+```sh
+sudo bash -c 'cat <<EOF > /etc/apt/sources.list.d/zabbly-incus-stable.sources
```

Comment:
There's no bashism in there, so `sh -c` would be preferable.

Reply:
changed in 0f054195bfe8618bcc25dab16e33ffa1307e4c6d